sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/HiveOrcQuerySuite.scala

```diff
@@ -210,7 +210,9 @@ class HiveOrcQuerySuite extends OrcQueryTest with TestHiveSingleton {
     }
   }
 
-  test("SPARK-23340 Empty float/double array columns raise EOFException") {
+  // SPARK-28885 String value is not allowed to be stored as numeric type with
```

Comment:
This query writes array not string.

Reply:
```
> spark.sql("select array()")
res0: org.apache.spark.sql.DataFrame = [array(): array<string>]
```

I think it makes more sense to make the type of empty array as `array<null>`.
How about create a jira for this and disable the test case for now?

Reply:
Created https://issues.apache.org/jira/browse/SPARK-29462 for this.